polkadot/primitives/src/vstaging/mod.rs

```diff
@@ -22,3 +22,14 @@ use bitvec::vec::BitVec;
 
 /// Bit indices in the `HostConfiguration.node_features` that correspond to different node features.
 pub type NodeFeatures = BitVec<u8, bitvec::order::Lsb0>;
+
+/// Module containing feature-specific bit indices into the `NodeFeatures` bitvec.
+pub mod node_features {
+	/// Tells if tranch0 assignments could be sent in a single certificate.
+	/// Reserved for: https://github.com/paritytech/polkadot-sdk/issues/628
+	pub const ENABLE_ASSIGNMENTS_V2: u8 = 0;
```

Comment:
Ok, so there will be a second flag for approval signature over multiple candidates. We could also group them under the same bit, but maybe it's safer to enable incrementally.

Reply:
1. I wouldn't group them under the same bit, because I don't think it would be a good idea to enable both at the same time, to minimise risks I would enable them a different point in time.
2. For approval signature over multiple candidates we won't use node_features, we will use this https://github.com/paritytech/polkadot-sdk/pull/1191/files#diff-f46e0114b0c7700357e785dcc1c542d2f61493122f23573fa9b25d6add814452R44, since I think it is a good idea to have `max_approval_coalesce_count` as a knob which we can fine tune depending on the network.